libs/ui/src/components/screen-components/header-container/components/screen-title/components/back-button/back-button.tsx

```diff
@@ -0,0 +1,17 @@
+import { OnEventFn } from '@rnw-community/shared';
+import React, { FC } from 'react';
+import { GestureResponderEvent } from 'react-native';
+
+import { TestIDProps } from '../../../../../../../interfaces/test-id.props';
+import { IconNameEnum } from '../../../../../../icon/icon-name.enum';
+import { TouchableIcon } from '../../../../../../touchable-icon/touchable-icon';
+
+import { styles } from './back-button.styles';
+
+interface Props extends TestIDProps {
+  onPress: OnEventFn<GestureResponderEvent>;
+}
+
+export const BackButton: FC<Props> = ({ onPress, testID }) => (
```

Comment:
what is the point of this component?  
could you reuse `HeaderBackButton`?

Reply:
no point, will revert changes